core/vibes/soul/primitives/product-card/index.tsx

```diff
@@ -55,7 +58,8 @@ export interface ProductCardProps {
  * ```
  */
 export function ProductCard({
-  product: { id, title, subtitle, badge, price, image, href, inventoryMessage },
+  product: { id, title, subtitle, badge, price, image, href, inventoryMessage, rating },
+  reviewsEnabled = false,
```

Comment:
Is `false` the right default here? I used `false` simply to match the default value for `showCompare` below. I guess I'm not entirely sure of all the scenarios in which GQL returns `null` for the settings field (that's the only reason that this prop would receive `undefined`)... and therefore am not sure if `false` is the best default.

Reply:
It's a tradeoff. `false` protects against accidental over-exposure of data in unexpected scenarios or failure scenarios, which seems like a better option than a fail-open. In practice we expect a value to always be returned.